### PR TITLE
Rename 'theme' to 'flavour' to keep consistency

### PR DESCRIPTION
**Pull request by kkrypt0nn**
Not quite sure why in the Go implementation `Theme` has been used instead of `Flavour` like other languages. Would be nice to keep consistency at least for languages libraries/implementations.

See [`Python`](https://github.com/catppuccin/python/blob/main/catppuccin.py#L26) and [`Rust`](https://github.com/catppuccin/rust/blob/main/src/flavour.rs#L4) as two among other examples.

### CODE REVIEW

File: _examples/catppuccin.tape [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Output catppuccin.gif
 
-Set Theme "Catppuccin Mocha"
+Set Flavour "Catppuccin Mocha"
```
Comment (caarlos0) [inline _examples/catppuccin.tape]: this is invalid syntax...

Comment (caarlos0) [inline _examples/catppuccin.tape]:
```suggestion
Set Theme "Catppuccin Mocha"
```

File: main.go [hunk 2]
```diff
@@ -2,8 +2,8 @@ package catppuccingo
 
 import "image/color"
 
-// Theme is an interface implemented by all Catppuccin variations.
-type Theme interface {
+// Flavour is an interface implemented by all Catppuccin variations.
+type Flavour interface {
```
Comment (caarlos0) [inline main.go:6]: should probably add a type alias to keep compatibility

File: _examples/catppuccin.tape (hunk 1 above)
Comment (kkrypt0nn) [reply _examples/catppuccin.tape]: Ah yeah right, my bad haven't seen that one ^^

**Comment by caarlos0** [comment]
Thanks!

**Pull request merged**